modules/search/class-jetpack-instant-search.php

```diff
@@ -119,11 +119,12 @@ protected function inject_javascript_options() {
 		$prefix  = Jetpack_Search_Options::OPTION_PREFIX;
 		$options = array(
 			'overlayOptions'        => array(
-				'colorTheme'      => get_option( $prefix . 'color_theme', 'light' ),
-				'enableInfScroll' => (bool) get_option( $prefix . 'inf_scroll', false ),
-				'highlightColor'  => get_option( $prefix . 'highlight_color', '#FFC' ),
-				'opacity'         => (int) get_option( $prefix . 'opacity', 97 ),
-				'showPoweredBy'   => (bool) get_option( $prefix . 'show_powered_by', true ),
+				'colorTheme'       => get_option( $prefix . 'color_theme', 'light' ),
+				'enableInfScroll'  => (bool) get_option( $prefix . 'inf_scroll', false ),
+				'enableTransition' => (bool) get_option( $prefix . 'overlay_transition', false ),
```

Comment:
Had to set this to default as `false` to get the class removed when the option was off, but it doesn't make sense to me if it's fetching the option. Could you give this a closer look @jsnmoon ? Thanks in advance.

Reply:
If we want this to default to `false`, I think we need set the control to default to `false` as well.

Reply:
I actually wanted it to default to `true`, but the class wasn't being removed when I checked the option off on the Customizer. This was the only way it worked for me, but maybe I did something wrong in my testing?

Reply:
Anyway, it might be easier (and better) to just cut off the Customizer option and go for the transition which is tiny at this point.